### DISCUSSION TITLE
Help needed with Linux Client. Server uses Let's Encrypt certs.

### DISCUSSION THREAD

**Discussion by papaarif-nb** [strongswan/strongswan#2521] [Q&A]
Hello.

I've configured a VPN server (or, rather jawj's bot did) with let's encrypt certs. While I can connect to the VPN using an Android or a Windows device, I can't connect to it using my Linux device. I'm using Nobara Linux 40, gnome edition.

Client software is strongSwan 5.9. I've configured an ipsec.conf (can share it if needed) and an ipsec.secrets files, but can't connect. Judging by the journal entries, the reason appear to be:
- StrongSwan has received the Let's Encrypt certificate, but it's reporting that it can't find the issuer of the certificate.
- There's no trusted RSA public key found for the server's certificate.

Please help me, or let me know if an issue isn't with the client, but may be with the server or the operating system itself.

**Comment by tobiasbrunner** [comment]
> StrongSwan has received the Let's Encrypt certificate, but it's reporting that it can't find the issuer of the certificate.

Then either the CA certificate (has to be locally installed and trusted) or the intermediate CA certificate (usually sent by the server) is missing. Check the log and your configuration.
